lighthouse-core/scripts/i18n/collect-strings.js

```diff
@@ -640,7 +641,7 @@ if (require.main === module) {
 
   if (collisions > 0) {
     console.log(`MEANING COLLISION: ${collisions} string(s) have the same content.`);
-    assert.equal(collisions, 30, `The number of duplicate strings have changed, update this assertion if that is expected, or reword strings. Collisions: ${collisionStrings.join('\n')}`);
+    assert.equal(collisions, 28, `The number of duplicate strings have changed, update this assertion if that is expected, or reword strings. Collisions: ${collisionStrings.join('\n')}`);
```

Comment:
idk why this changed

Reply:
that's somewhat concerning... :/

can we do a diff?

this is what I see on master

```
    "When an element doesn't have an accessible name, screen readers announce it with a generic name, making it unusable for users who rely on screen readers. $LINK_START_0$Learn more$LINK_END_0$.",
    "When an element doesn't have an accessible name, screen readers announce it with a generic name, making it unusable for users who rely on screen readers. $LINK_START_0$Learn more$LINK_END_0$.",
    'ARIA $MARKDOWN_SNIPPET_0$ elements have accessible names',
    'ARIA $MARKDOWN_SNIPPET_0$ elements have accessible names',
    'ARIA $MARKDOWN_SNIPPET_0$ elements do not have accessible names.',
    'ARIA $MARKDOWN_SNIPPET_0$ elements do not have accessible names.',
    "When an element doesn't have an accessible name, screen readers announce it with a generic name, making it unusable for users who rely on screen readers. $LINK_START_0$Learn more$LINK_END_0$.",
    'ARIA $MARKDOWN_SNIPPET_0$ elements have accessible names',
    'ARIA $MARKDOWN_SNIPPET_0$ elements do not have accessible names.',
    "When an element doesn't have an accessible name, screen readers announce it with a generic name, making it unusable for users who rely on screen readers. $LINK_START_0$Learn more$LINK_END_0$.",
    'ARIA $MARKDOWN_SNIPPET_0$ elements have accessible names',
    'ARIA $MARKDOWN_SNIPPET_0$ elements do not have accessible names.',
    "When an element doesn't have an accessible name, screen readers announce it with a generic name, making it unusable for users who rely on screen readers. $LINK_START_0$Learn more$LINK_END_0$.",
    '$MARKDOWN_SNIPPET_0$ elements have $MARKDOWN_SNIPPET_1$ text',
    '$MARKDOWN_SNIPPET_0$ elements have $MARKDOWN_SNIPPET_1$ text',
    '$MARKDOWN_SNIPPET_0$ elements do not have $MARKDOWN_SNIPPET_1$ text',
    '$MARKDOWN_SNIPPET_0$ elements do not have $MARKDOWN_SNIPPET_1$ text',
    'Document has a valid $MARKDOWN_SNIPPET_0$',
    'Document has a valid $MARKDOWN_SNIPPET_0$',
    'Potential Savings',
    'Potential Savings',
    'Failing Elements',
    'Failing Elements',
    'URL',
    'URL',
    'Consider uploading your GIF to a service which will make it available to embed as an HTML5 video.',
    'Consider uploading your GIF to a service which will make it available to embed as an HTML5 video.',
    'Consider using a $LINK_START_0$plugin$LINK_END_0$ or service that will automatically convert your uploaded images to the optimal formats.',
    'Consider using a $LINK_START_0$plugin$LINK_END_0$ or service that will automatically convert your uploaded images to the optimal formats.',
    'Consider uploading your GIF to a service which will make it available to embed as an HTML5 video.'
```

Reply:
```
"When an element doesn't have an accessible name, screen readers announce it with a generic name, making it unusable for users who rely on screen readers. $LINK_START_0$Learn more$LINK_END_0$.",
'ARIA $MARKDOWN_SNIPPET_0$ elements have accessible names',
'ARIA $MARKDOWN_SNIPPET_0$ elements do not have accessible names.',
"When an element doesn't have an accessible name, screen readers announce it with a generic name, making it unusable for users who rely on screen readers. $LINK_START_0$Learn more$LINK_END_0$.",
'ARIA $MARKDOWN_SNIPPET_0$ elements have accessible names',
'ARIA $MARKDOWN_SNIPPET_0$ elements do not have accessible names.',
"When an element doesn't have an accessible name, screen readers announce it with a generic name, making it unusable for users who rely on screen readers. $LINK_START_0$Learn more$LINK_END_0$.",
'ARIA $MARKDOWN_SNIPPET_0$ elements have accessible names',
'ARIA $MARKDOWN_SNIPPET_0$ elements do not have accessible names.',
"When an element doesn't have an accessible name, screen readers announce it with a generic name, making it unusable for users who rely on screen readers. $LINK_START_0$Learn more$LINK_END_0$.",
'$MARKDOWN_SNIPPET_0$ elements have $MARKDOWN_SNIPPET_1$ text',
'$MARKDOWN_SNIPPET_0$ elements do not have $MARKDOWN_SNIPPET_1$ text',
'Document has a valid $MARKDOWN_SNIPPET_0$',
'Potential Savings',
'Failing Elements',
'Consider uploading your GIF to a service which will make it available to embed as an HTML5 video.',
'Consider using a $LINK_START_0$plugin$LINK_END_0$ or service that will automatically convert your uploaded images to the optimal formats.',
'Consider uploading your GIF to a service which will make it available to embed as an HTML5 video.'
```

diff https://www.diffchecker.com/TFvWBE9j

Reply:
hm, that's 10 more missing than there should be, I had to add `collisionStrings.push` where the first `collisions++` is

Reply:
oh.. then:

![image](https://user-images.githubusercontent.com/4071474/123339745-fcebe880-d4ff-11eb-8220-1369680b60ca.png)

Reply:
`collectAllStringsInDir` does have some local data per directory, and renderer strings moved to a new folder, so that must be why. no problem here i guess ... except maybe in the "collision detection" code.

Reply:
https://github.com/GoogleChrome/lighthouse/pull/12697

Reply:
I didn't follow up after https://github.com/GoogleChrome/lighthouse/pull/12441/files#r630521367 :(